package.json

```diff
@@ -50,6 +52,7 @@
     "@types/node": "22.15.29",
     "license-checker": "25.0.1",
     "shx": "0.4.0",
+    "ts-standard": "^12.0.2",
```

Comment:
Note that ts-standard is stuck at eslint 8 - to use eslint 9 you need to use something like neostandard.

Reply:
good catch. I'll fix this soon. mostly was trying to stay consistent with the client, which still uses `ts-standard` (for now)